### PR TITLE
Remove gpg keys from git-crypt

### PR DESCRIPTION
**Pull request by dhirajnarwanimoj**
* Create a script to remove unused gpg keys from git-crypt
* Ran it for users who don't need access any more. NB This rewrites all commits, to remove their .gpg file, hence why it looks like all files are touched.
* Add key 21070F2CABCC1398159554E3F8379AB175E3131C.gpg

(The repo's symmetric key is not rotated in this PR - see next PR)

### CODE REVIEW

File: remove-gpg-keys.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+set -ex
+
+KEY_FILE=$PWD/keys.txt
```
Comment (xoen) [inline remove-gpg-keys.sh:5]: the `keys.txt` file has the list of keys to delete right?

What is the `git filter-branch -f --index-filter "git rm --cached --ignore-unmatch .git-crypt/keys/default/0/$key" HEAD` command does? Sorry not that familiar with rewriting Git history like this. 
Don't know how it works or if it's worth adding a comment to elaborate on what these 2 Git command would do?

Comment (dhirajnarwanimoj) [reply remove-gpg-keys.sh:5]: This scans all the commits made to that file and filepath and removes them.

Comment (dhirajnarwanimoj) [reply remove-gpg-keys.sh:5]: Done

**Comment by xoen** [comment]
@ntoll you mentioned you have a fair amount of experience with GPG, I wonder if you also have some experience with `git-crypt`, maybe you want to have a look a this PR as well as my knowledge of this is very limited.

**Comment by davidread** [comment]
I suggest we ask him to review the next one

**Comment by davidread** [comment]
This PR failed. We think the history was not rewritten due to pressing the GitHub "resolve conflicts" button.

**Pull request merged**